core/src/main/scala/akka/kafka/scaladsl/Consumer.scala

```diff
@@ -171,5 +172,9 @@ object Consumer {
     }
   }
 
+  def committablePartitionedSource[K, V](settings: ConsumerSettings[K, V]):
+    Source[(TopicPartition, Source[CommittableMessage[K, V], Control]), Control] = {
```

Comment:
Some discussion is in the docs: https://github.com/akka/akka/pull/19501/files
and the PR that made them SubFlow https://github.com/akka/akka/issues/19005
Source of Source is inherently a bit dangerous - one needs to make sure to never drop one etc (and other reasons, like gearpump not being able to do that)